src/opers.c

```diff
@@ -4394,6 +4394,7 @@ static Int InitLibrary (
     SET_LEN_PLIST(HIDDEN_IMPS, 0);
     WITH_HIDDEN_IMPS_FLAGS_CACHE = NEW_PLIST(T_PLIST, HIDDEN_IMPS_CACHE_LENGTH * 2);
     SET_LEN_PLIST(WITH_HIDDEN_IMPS_FLAGS_CACHE, HIDDEN_IMPS_CACHE_LENGTH * 2);
+    AssGVar(GVarName("HIDDEN_IMPS"), HIDDEN_IMPS);
```

Comment:
Why is this needed?

Reply:
To be able to see the content of `HIDDEN_IMPS` from GAP level.